modules/landing-zone-bootstrap/locals.tf

```diff
@@ -25,12 +25,12 @@ locals {
     "roles/serviceusage.serviceUsageAdmin",
     "roles/resourcemanager.projectDeleter",
     "roles/viewer",
+    "roles/iam.serviceAccountTokenCreator",
```

Comment:
nice one - odd that we are working without the saTC
perhaps the admin above is superceeding
   "roles/iam.serviceAccountAdmin",